### PR TITLE
fix existing 'inbetween' in dictionary.txt

### PR DESCRIPTION
**Pull request by luzpaz**
inbetween->in between
src: https://ell.stackexchange.com/questions/63622/should-i-use-in-between-or-inbetween-in-the-following-sentence

### CODE REVIEW

**Comment by luzpaz** [comment]
@Eric89GXL ping

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -2344,7 +2344,7 @@ inappropiate->inappropriate
 inaugures->inaugurates
 inbalance->imbalance
 inbalanced->imbalanced
-inbetween->between
+inbetween->in between
```
Comment (Eric89GXL) [inline codespell_lib/data/dictionary.txt]: perhaps both should be offered

Comment (lucasdemarchi) [inline codespell_lib/data/dictionary.txt]: I'm not sure about this one. I've seen disagreements on dictionaries about it in the past. grammarly.com says it's incorrect, but it does appear on some dictionaries as "informally accepted".

Comment (Eric89GXL) [inline codespell_lib/data/dictionary.txt]: My vote would be to recommend "between" (by listing it first) but allow "in between" as second option

**Comment by luzpaz** [comment]
resubmit PR

**Comment by luzpaz** [comment]
soft bump

**Pull request merged**